tests/regression/41-stdlib/06-atomic.c

```diff
@@ -0,0 +1,11 @@
+int g = 8;
+
+int main() {
+    int i;
+    __atomic_store_n (&i, 12, __ATOMIC_RELAXED);
+    int i = __atomic_load_n (&i, __ATOMIC_RELAXED);
```

Comment:
This doesn't even compile because of the variable redefinition in the same block.